firefox-ios/Client/Frontend/Settings/SettingsTableViewController.swift

```diff
@@ -746,6 +725,32 @@ class CheckmarkSetting: Setting {
         }
     }
 
+    private func setupLeftCheckLabel(_ cell: UITableViewCell, theme: Theme) {
+        check.font = FXFontStyles.Regular.body.scaledFont()
```

Comment:
Judging by the font size we had for the checkmark before shouldn't the font be `title3` instead of `body`?

Reply:
Yes you are right is `title3`